### PR TITLE
Add Ukrainian to docsbuild

### PR DESCRIPTION
**Pull request by kazanzhy**

### CODE REVIEW

File: build_docs.py [hunk 1]
```diff
@@ -295,6 +295,7 @@ def __gt__(self, other):
     Language("zh-cn", "zh_CN", "Simplified Chinese", True, XELATEX_WITH_CJK),
     Language("zh-tw", "zh_TW", "Traditional Chinese", True, XELATEX_WITH_CJK),
     Language("pl", "pl", "Polish", False, XELATEX_DEFAULT),
+    Language("uk", "uk", "Ukrainian", True, XELATEX_DEFAULT),
```
Comment (m-aciek) [inline build_docs.py:298]:
```suggestion
    Language("uk", "uk", "Ukrainian", False, XELATEX_DEFAULT),
```
`in_prod` is responsible for displaying in language switcher, which according to https://peps.python.org/pep-0545/#add-support-for-translations-in-docsbuild-scripts should be done after having those priority resources translated.

Comment (kazanzhy) [reply build_docs.py:298]: Thanks for your suggestions.
These documents are already translated and are reviewing now.

Comment (m-aciek) [reply build_docs.py:298]: Wow, congrats! Wasn't expecting that :)

**Comment by JulienPalard** [comment]
@kazanzhy docsbuild-scripts is a cron task running daily, so don't worry if it take some time to show up.

**Pull request merged**